<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CityDrain3</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../data/resources/cd3-gui.qrc">
    <normaloff>:/icons/logo3.png</normaloff>:/icons/logo3.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::RubberBandDrag</enum>
      </property>
      <property name="cacheMode">
       <set>QGraphicsView::CacheNone</set>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
      </property>
      <property name="rubberBandSelectionMode">
       <enum>Qt::IntersectsItemBoundingRect</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QProgressBar" name="simProgressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;run</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/resources/cd3-gui.qrc">
          <normaloff>:/icons/media-playback-start.png</normaloff>:/icons/media-playback-start.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;stop</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/resources/cd3-gui.qrc">
          <normaloff>:/icons/process-stop.png</normaloff>:/icons/process-stop.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Simulation</string>
    </property>
    <addaction name="actionNewSimulation"/>
    <addaction name="actionSave_Simulation"/>
    <addaction name="actionSave_as"/>
    <addaction name="action_open"/>
    <addaction name="actionClose"/>
    <addaction name="actionExport_to_pdf"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Plugin"/>
    <addaction name="actionAdd_Python_Module"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="action_delete"/>
    <addaction name="actionSelect_all"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionFind_node"/>
    <addaction name="actionRename"/>
    <addaction name="separator"/>
    <addaction name="actionHorizontal_center_align"/>
    <addaction name="actionAlign_vertically_center"/>
    <addaction name="actionIncrease_distance"/>
    <addaction name="actionDecrease_distance"/>
    <addaction name="actionAuto_layout"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewSimulation"/>
   <addaction name="actionSave_Simulation"/>
   <addaction name="action_open"/>
   <addaction name="action_exit"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_out"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_reset"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="logWidget">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="maximumBlockCount">
        <number>50000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAdd_Plugin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
   </property>
   <property name="text">
    <string>add Plugin</string>
   </property>
   <property name="toolTip">
    <string>add a native plugin to the project</string>
   </property>
  </action>
  <action name="actionNewSimulation">
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/document-new.png</normaloff>:/icons/document-new.png</iconset>
   </property>
   <property name="text">
    <string>new</string>
   </property>
   <property name="toolTip">
    <string>Create a new Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave_Simulation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/document-save.png</normaloff>:/icons/document-save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>Save the current Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
   <property name="toolTip">
    <string>Open a Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAdd_Python_Module">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
   </property>
   <property name="text">
    <string>add Python Module</string>
   </property>
   <property name="toolTip">
    <string>add a Python Module to the Project</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/application-exit.png</normaloff>:/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;exit</string>
   </property>
   <property name="toolTip">
    <string>exit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/zoom-in.png</normaloff>:/icons/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/zoom-out.png</normaloff>:/icons/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_reset">
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/zoom-original.png</normaloff>:/icons/zoom-original.png</iconset>
   </property>
   <property name="text">
    <string>zoom_reset</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/document-save-as.png</normaloff>:/icons/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExport_to_pdf">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>export to pdf</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionFind_node">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/edit-find.png</normaloff>:/icons/edit-find.png</iconset>
   </property>
   <property name="text">
    <string>find node</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionHorizontal_center_align">
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/stock_graphics-align-top.png</normaloff>:/icons/stock_graphics-align-top.png</iconset>
   </property>
   <property name="text">
    <string>horizontal center align</string>
   </property>
  </action>
  <action name="actionAlign_vertically_center">
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/stock_graphics-align-left.png</normaloff>:/icons/stock_graphics-align-left.png</iconset>
   </property>
   <property name="text">
    <string>vertical center align</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>&amp;delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/edit-undo.png</normaloff>:/icons/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/resources/cd3-gui.qrc">
     <normaloff>:/icons/edit-redo.png</normaloff>:/icons/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionIncrease_distance">
   <property name="text">
    <string>increase distance</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionDecrease_distance">
   <property name="text">
    <string>decrease distance</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAuto_layout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>auto layout</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../data/resources/cd3-gui.qrc"/>
 </resources>
 <connections/>
</ui>
